docker/check-cpu.sh

```diff
@@ -10,7 +10,7 @@
 
 NPROC=$(nproc)
 
-FREE_RAM=$(awk '/^MemAvailable:/{printf("%d",$2/1024/1024)}' /proc/meminfo)
+FREE_MEM=$(free -m | awk '/^Mem:/{printf("%.0f\n",$2/1000)}')
```

Comment:
maybe better to restore the previous code:
https://github.com/alicevision/AliceVision/commit/ad13cf380c498279195f8d0d2bab8c65142decf9